daprdocs/content/en/getting-started/get-started-api.md

```diff
@@ -3,27 +3,40 @@ type: docs
 title: "Use the Dapr API"
 linkTitle: "Use the Dapr API"
 weight: 30
+description: "Run a Dapr sidecar and try out the state API"
 ---
 
-After running the `dapr init` command in the [previous step]({{<ref install-dapr-selfhost.md>}}), your local environment has the Dapr sidecar binaries as well as default component definitions for both state management and a message broker (both using Redis). You can now try out some of what Dapr has to offer by using the Dapr CLI to run a Dapr sidecar and try out the state API that will allow you to store and retrieve a state. You can learn more about the state building block and how it works in [these docs]({{< ref state-management >}}).
+Running [`dapr init`]({{<ref install-dapr-selfhost.md>}}) loads your local environment with:
 
-You will now run the sidecar and call the API directly (simulating what an application would do).
+- The Dapr sidecar binaries.
+- Default Redis component definitions for both:
+  - State management, and
+  - A message broker.
 
-## Step 1: Run the Dapr sidecar
+With this setup, you can run a Dapr sidecar with the Dapr CLI and try out the state API to store and retrieve a state. [Learn more about the state building block and how it works in our concept docs]({{< ref state-management >}}).
```

Comment:
```suggestion
With this setup, you run Dapr using the Dapr CLI and try out the state API to store and retrieve a state. [Learn more about the state building block and how it works in our concept docs]({{< ref state-management >}}).
```
I think we often just say running Dapr unless we really need to talk about the sidecar architecture.  Also, I think we should use present tense and confident wording as consistently possible.  "you can run" or "you will run" changes to "you run".